js/common/lib/env.ts

```diff
@@ -11,6 +11,11 @@ export declare namespace Env {
      */
     numThreads?: number;
 
+    /**
+     * set a logging level. If omitted, default is 'warning'
+     */
+    loggingLevel?: 'verbose'|'info'|'warning'|'error'|'fatal';
```

Comment:
considering #7677, there is requirement for supporting logging level options in a higher scope.

currently this is `ort.env.wasm.loggingLevel`. should we move it to `ort.env.loggingLevel`?